src/Suave/Web.fs

```diff
@@ -43,7 +43,7 @@ module Web =
     let toRuntime = SuaveConfig.toRuntime config homeFolder compressionFolder
 
     if config.initialiseLogger then
-      Global.initialise { Global.DefaultConfig with getLogger = fun _ -> config.logger }
+      Global.initialise { Global.defaultConfig with getLogger = fun _ -> config.logger }
```

Comment:
Let's use the [`initialiseIfDefault`](https://github.com/SuaveIO/suave/blob/f1a059f1d3a5b8eefd9c2ea0b24ee8eb4c275344/paket-files/logary/logary/src/Logary.Facade/Facade.fs#L818) which will automatically check if logging has already been set by the composition root/caller.